### DISCUSSION TITLE
Feature Request: Remove the Bits Counter next to the Channel Points in the new Power-ups Update

### DISCUSSION THREAD

**Discussion by lukasf6** [night/betterttv#6880] [Ideas]
Feature Request: Remove the Bits Counter next to the Channel Points in the new Power-ups Update
Also consider disabling all the features when used by other users

**Comment by bubbabdfjhgldkfhg** [comment]
> consider disabling all the features when used by other users

please god let me turn it all off

**Comment by mparmpuni** [comment]
Yes please. Those big emotes are annoying. I used UblockOrigin to remove the other effects but I can't make the emotes small again unless I refresh the page.

**Comment by Lling** [comment]
To disable the whole power up bs would be awesome.

**Comment by xBlue555** [comment]
I also hate the power up crap. removing all of it would be nice. thanks

**Comment by night** [comment]
The Hide Bits setting (Settings → Chat → Bits) hides the bits button and counter. Closing as resolved; please open a new discussion if the Power-ups update introduced a separate element this doesn't cover.
